### PR TITLE
Add DevCon24 banner

### PR DESCRIPTION
**Pull request by pedrominatel**
This PR removes the Registry banner and adds the DevCon24 event logo and information.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -7,17 +7,22 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
...
-## Project Highlight
+## Upcoming Event: DevCon24
 
 <p align="center">
-    <img src="esp_registry_logo.png">
+    <img src="devcon24.png">
 </p>
   
-###  **ESP Component Registry**
+Get excited and stay tuned for talks on various presentation topics, new product releases, and upgraded solutions at the Espressif Developer Conference!
```
Comment (VojtechBartoska) [inline profile/README.md:16]: I will suggest to change to:

Get excited and stay tuned for various talks and Speakers, new product releases, and upgraded solutions at the Espressif Developer Conference!

File: profile/README.md [hunk 2]
```diff
@@ -7,17 +7,22 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
...
 
-The **[ESP Component Registry](https://components.espressif.com/)** allows you to discover, download, and publish components and examples for the ESP-IDF. 
+### Join Us!
 
-If you have any issues or suggestions about the ESP Component Registry, feel free to open an issue in the [IDF Component Manager repository](https://github.com/espressif/idf-component-manager).  
+- **When: Sep 3-5, 13:00-18:30 CEST**
+- **Where: [Espressif YouTube Channel](https://www.youtube.com/@EspressifSystems)**
+
+We are thrilled to have you join our community of innovators and creators in the exciting world of Espressif products! The Espressif Developer Portal is not just a platform; it’s a community-driven space where ideas flourish and projects come to life.
```
Comment (VojtechBartoska) [inline profile/README.md:23]: this line can be deleted, probably it is some leftover from Dev Portal?

File: profile/README.md [hunk 3]
```diff
@@ -7,17 +7,22 @@ To check out all the series of SoCs from Espressif, please visit our [ESP Produc
...
+
+For more information about it, please visit [DevCon24](https://devcon.espressif.com/) site.
```
Comment (VojtechBartoska) [inline profile/README.md:25]: Text change + bold formatting.

** For more information and full conference program, please visit [DevCon24 website](https://devcon.espressif.com/).**

**Review by VojtechBartoska — CHANGES_REQUESTED** [review]
few content and formatting changes

**Comment by pedrominatel** [comment]
Done, @VojtechBartoska. Thank you for your review.

**Comment by VojtechBartoska** [comment]
Feel free to merge @pedrominatel

**Pull request merged**